ui/app/components/clients/horizontal-bar-chart.js

```diff
@@ -218,11 +195,6 @@ export default class HorizontalBarChart extends Component {
         } else {
           this.tooltipTarget = null;
         }
-        dataBars
```

Comment:
discussed with design that we no longer need to change the bar color on hover, the grey bar background is sufficient
<img width="697" alt="Screenshot 2024-01-17 at 11 49 25 AM" src="https://github.com/hashicorp/vault/assets/68122737/6d1a0db2-adf2-4eaf-aa77-71f059dedb20">